base/setup/usetup/CMakeLists.txt

```diff
@@ -42,5 +42,5 @@ endif()
 add_pch(usetup usetup.h SOURCE)
 set_module_type(usetup nativecui)
 target_link_libraries(usetup inflib setuplib zlib_solo ext2lib vfatlib btrfslib)
-add_importlibs(usetup ntdll)
+add_importlibs(usetup kernel32 ntdll)
```

Comment:
This will not work, usetup is a native NT application and cannot import kernel32.

Reply:
OK.